extensions/ql-vscode/src/view/method-modeling/ModeledMethodAlert.tsx

```diff
@@ -6,12 +6,12 @@ import { useCallback } from "react";
 
 type Props = {
   error: ModeledMethodValidationError;
-  setSelectedIndex: (index: number) => void;
+  setSelectedIndex?: (index: number) => void;
```

Comment:
It would be nice if we were able to highlight the model on which the error is located, but this can still be added in a follow-up PR if this is something that comes out of a design review.

Reply:
I've opened a separate issue for this so we don't forget it.